plugins/chat/assets/javascripts/discourse/models/chat-message-reaction.js

```diff
@@ -0,0 +1,33 @@
+import { tracked } from "@glimmer/tracking";
+import User from "discourse/models/user";
+import { TrackedArray } from "@ember-compat/tracked-built-ins";
+
+export default class ChatMessageReaction {
+  static create(args = {}) {
+    return new ChatMessageReaction(args);
+  }
+
+  @tracked count = 0;
+  @tracked reacted = false;
+  @tracked users = [];
+
+  constructor(args = {}) {
```

Comment:
Same destructuring could be done here (with the exception of `initUserModels`)

Reply:
Yes same reason than above, not super fan of destructuring in things which can be long lists